backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -139,6 +139,10 @@ span.info {
 .field {
   padding: 10px 0;
 
+  &.centered {
+    margin-top: 13px;
+  }
```

Comment:
I don't like this, but if I don't do something like that the checkbox that it's applied to ("Track inventory") is way too far up. I'm sure there's a better way to do this. Frontenders?